api/graphql/validation_errors.py

```diff
@@ -43,6 +43,7 @@ class ValidationErrorCodes(Enum):
     INVALID_PAYMENT_TYPE = "INVALID_PAYMENT_TYPE"
     MISSING_PAYMENT_PRODUCT = "MISSING_PAYMENT_PRODUCT"
     UPSTREAM_CALL_FAILED = "UPSTREAM_CALL_FAILED"
+    DELETION_NOT_ALLOWED = "DELETION_NOT_ALLOWED"
```

Comment:
This is not used anywhere, probably because `ValidationErrorWithCode` is not used ☺️

Reply:
Yup, I was implementing this with ``ValidationErrorWithCode`` but as mentioned above decided not to.

Reply:
I guess this line could be removed then to get rid of the unused code.

Side note: searching and removing unused code could be fun Friday activity in general. I've found few of them just by luck so I think our static code analyzers don't inspect that 🤔 

Perhaps something for Friyays 😄 

Other than that, the PR looks great!

Reply:
its now removed.